client/src/index.css

```diff
@@ -1,5 +1,152 @@
+@import url('../node_modules/photon-colors/photon-colors.css');
+
+html,
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--grey-10-a60);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin: 0;
+}
+
+/* Photon Components https://github.com/FirefoxUX/photon-components-web/blob/master/photon-buttons/button.css */
```

Comment:
We should probably just pull a copy of this file into the tree and `@import` it like we do for the photon-colors. 🙂